lib/generators/lets_encrypt/install_generator.rb

```diff
@@ -21,6 +21,11 @@ def copy_migrations
                            'db/migrate/create_letsencrypt_certificates.rb'
       end
 
+      def copy_initializer
+        migration_template 'letsencrypt.rb',
+                           'config/initializers/letsencrypt.rb'
```

Comment:
The `migration_template` is designed for migration. You can use `copy_file` for the correct action.

https://guides.rubyonrails.org/generators.html#generator-helper-methods

Reply:
Renaming `copy_initializer` to `copy_config` will be clean for this action.